index.js

```diff
@@ -0,0 +1,17 @@
+function fibonacci(number) {
```

Comment:
you can also have function name as `printFibonacciNumbers`  which sounds more better